rdkit/build.sh

```diff
@@ -28,11 +27,9 @@ cmake \
 if [[ `uname` == 'Linux' ]]; then
     make -j$CPU_COUNT 
     RDBASE=$SRC_DIR LD_LIBRARY_PATH="$PREFIX/lib:$SRC_DIR/lib" PYTHONPATH=$SRC_DIR ctest -j$CPU_COUNT --output-on-failure
-    RDBASE=$SRC_DIR LD_LIBRARY_PATH="$PREFIX/lib:$SRC_DIR/lib" PYTHONPATH=$SRC_DIR $PYTHON "$RECIPE_DIR/pkg_version.py"
 else
     make -j$CPU_COUNT install
-    RDBASE=$SRC_DIR DYLD_FALLBACK_LIBRARY_PATH="$PREFIX/lib:$SRC_DIR/lib" PYTHONPATH=$SRC_DIR ctest -j$CPU_COUNT --output-on-failure
-    RDBASE=$SRC_DIR DYLD_FALLBACK_LIBRARY_PATH="$PREFIX/lib:$SRC_DIR/lib" PYTHONPATH=$SRC_DIR $PYTHON "$RECIPE_DIR/pkg_version.py"
+    RDBASE=$SRC_DIR DYLD_FALLBACK_LIBRARY_PATH="$PREFIX/lib:$SRC_DIR/lib:/usr/lib" PYTHONPATH=$SRC_DIR ctest -j$CPU_COUNT --output-on-failure
```

Comment:
I had to add /usr/lib here for some strange reason to pick up libc++.   I think this is a safe fix but I can revert if necessary.